code/_onclick/hud/plane_master.dm

```diff
@@ -4,19 +4,9 @@
 	appearance_flags = PLANE_MASTER|NO_CLIENT_COLOR
 	blend_mode = BLEND_OVERLAY
 
-/obj/screen/plane_master/New()
-	// Yes because of course it's automatically implied that if a plane master uses blend multiply it ALWAYS needs a backdrop.
-	// Jesus christ this is 2012 SVN tier retardedness get your shit together TG.
-	/*
-	if(blend_mode == BLEND_MULTIPLY)
-		//What is this? Read http://www.byond.com/forum/?post=2141928
-		var/image/backdrop = image('icons/mob/screen_gen.dmi', "black")
-		backdrop.transform = matrix(200, 0, 0, 0, 200, 0)
-		backdrop.layer = BACKGROUND_LAYER
-		backdrop.blend_mode = BLEND_OVERLAY
-		add_overlay(backdrop)
-	*/
-	..()
+//Why do plane masters need a backdrop sometimes? Read http://www.byond.com/forum/?post=2141928
+//Trust me, you need one. Period. If you don't think you do, you're doing something extremely wrong.
```

Comment:
Nice meme m8, come back to argue about this when you understand what you're talking about.